### DISCUSSION TITLE
ccf-validation-tools/Urinary_bladder/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#279] [General]
# ccf-validation-tools/Urinary_bladder/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Urinary_bladder/comments/

**Comment by emquardokus** [comment]
Urinary_Bladder_v1.1_DRAFT
https://docs.google.com/spreadsheets/d/1iCZpti7fYupWhQjDz_tE01ii2WH23hIno9kYggMjDZo/edit#gid=1057183099

**Blank ontology ID missing parent*
Rows 29-45 which were recently added by author are due to a shift in columns such that the CT/1 labeled column is actually containing biomarkers.
Example: 

Contacting author Doug Strand for review of edited table.

**Comment by aleixpuigb** [comment]
New terms in ontology:
Urinary bladder : lateral wall of urinary bladder (UBERON:8600043)
Urinary bladder : left lateral wall of urinary bladder (UBERON:8600044)
Urinary bladder : right lateral wall of urinary bladder (UBERON:8600045)
Urinary bladder : anterior wall of urinary bladder (UBERON:8600046)
Urinary bladder : posterior wall of urinary bladder (UBERON:8600047)
Urinary bladder : apex of urinary bladder (UBERON:8600048)
Urinary bladder : dome of urinary bladder (UBERON:8600049)
